lib/active_merchant/billing/gateways/flex_charge.rb

```diff
@@ -68,7 +69,9 @@ def refund(money, authorization, options = {})
       end
 
       def void(money, authorization, options = {})
-        refund(money, authorization, options)
+        order_id, _currency = authorization.split('#')
+        self.money_format = :dollars
```

Comment:
Do you want to set this at the class level?

Reply:
is no needed, didn't commit my latest change